Formula/ant.rb

```diff
@@ -15,8 +15,8 @@ class Ant < Formula
   end
 
   resource "bcel" do
-    url "https://www.apache.org/dyn/closer.cgi?path=commons/bcel/binaries/bcel-6.3.1-bin.tar.gz"
```

Comment:
I think this does work, even though you got redirected to some mirror link.
`https://www.apache.org/dyn/closer.cgi?path=commons/bcel/binaries/bcel-6.4.0-bin.tar.gz`

Reply:
This works for the most recent version of the package. When the version is updated it is no longer available in the closer-redirected mirrors.